### DISCUSSION TITLE
Scdaemon相关

### DISCUSSION THREAD

**Discussion by hanyuwei70** [canokeys/canokey-product#13] [General]
碰到scdaemon各种发疯可以考虑在 scdaemon.conf 里面加上 `pcsc-shared` 缓解问题
另外考虑Key自身的问题
目前我自己的环境是 Win10 + GnuPG 2.3.4，实测 Canokey Piegon 和 Yubikey 5 没有问题，Yubikey neo有问题

**Comment by CoelacanthusHex** [comment]
根据 gpg 文档的描述，这个选项会破坏 scdaemon 的假设（即 scdaemon 独占），所以如果遇到 gpg 失灵可能需要考虑再关掉这个选项
